pyproject.toml

```diff
@@ -8,25 +8,68 @@ authors = [
 ]
 requires-python = ">=3.8.0"
 license = { text = "VSL" }
-dynamic = ["dependencies", "optional-dependencies", "version"]
+dynamic = ["version"]
+
+dependencies = [
+    "pefile>=2024.8.26",
+]
+
+[project.optional-dependencies]
+full = [
+    "yara-python>=4.5.1,<5",
+    "capstone>=5.0.3,<6",
+    "pycryptodome>=3.21.0,<4",
+    "leechcorepyc>=2.19.2,<3; sys_platform != 'darwin'",
+]
+
+cloud = [
```

Comment:
Feels  like this should be part of full, shouldn't it?  I can't see a reason for selecting full and then not getting cloud as well?